uast/uast-kotlin-fir/src/org/jetbrains/uast/kotlin/internal/firKotlinInternalUastUtils.kt

```diff
@@ -33,6 +37,19 @@ internal fun KtType.toPsiClass(context: KtElement): PsiClass? {
     }
 }
 
+internal fun KtCallElement.toFunctionLikeSymbol(): KtFunctionLikeSymbol? {
+    analyseForUast(this) {
+        return resolveCall()?.targetFunction?.candidates?.singleOrNull()
+    }
+}
+
+internal fun KtCallElement.toAnnotationConstructorSymbol(): KtConstructorSymbol? {
```

Comment:
The same as in https://github.com/JetBrains/intellij-kotlin/pull/96#discussion_r684035608 . `KtSymbol` should not be leaked outside `analyse` call (2)

Reply:
Ditto: dropped.